pkg/yurtctl/cmd/revert/edgenode.go

```diff
@@ -310,3 +318,7 @@ func (r *RevertEdgeNodeOptions) getKubeletSvcBackup() string {
 func (r *RevertEdgeNodeOptions) getYurthubYaml() string {
 	return filepath.Join(r.PodMainfestPath, enutil.YurthubYamlName)
 }
+
+func (r *RevertEdgeNodeOptions) getYurthubConf() string {
+	return filepath.Join(hubself.HubRootDir, hubself.HubName)
```

Comment:
Var HubRootDir is not found in hubself package

Reply:
```
 func (r *RevertEdgeNodeOptions) getYurthubConf() string {
-       return filepath.Join(hubself.HubRootDir, hubself.HubName)
+       return filepath.Join(hubself.hubRootDir, hubself.hubName)
 }
```

Sorry, my mistake.